src/frontend/App/hooks/useHentAlleGjenbrukbareVilkårsvurderinger.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback, useState } from 'react';
+import { RessursStatus } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+
+interface Props {
+    gjenbrukbareVilkårsvurderinger: string[];
+    hentAlleGjenbrukbareVilkårsvurderinger: (behandlingId: string) => Promise<void>;
+}
+
+export const useHentAlleGjenbrukbareVilkårsvurderinger = (): Props => {
+    const [gjenbrukbareVilkårsvurderinger, settGjenbrukbareVilkårsvurderinger] = useState<string[]>(
+        []
+    );
+
+    const { axiosRequest } = useApp();
+
+    const hentAlleGjenbrukbareVilkårsvurderinger = useCallback(
+        async (behandlingId: string): Promise<void> => {
```

Comment:
```suggestion
        async (behandlingId: string): Promise<void> => {
```
Hvorfor er dette satt opp som en async 

funksjon som skal returnere en Promise når vi aldri lytter på utfallet av promiset den returnerer? Nå klager for eksempel IDEen min på linje 143 i `
useVilkår` som vist under:

![Skjermbilde 2024-12-10 kl  13 18 12](https://github.com/user-attachments/assets/4948eaea-efbb-4ab9-b744-2ae24f8fdf35)

Jeg ville derfor fjernet async-notasjonen

Reply:
Endret